### PR TITLE
wlan(4): minor improvements

### PR DESCRIPTION
**Pull request by tomhukins**

### CODE REVIEW

**Comment by emaste** [comment]
I think Wi-Fi or WiFi is more common than Wifi, and Google Trends suggests no hyphen is more common
https://trends.google.ca/trends/explore?date=now%201-d&geo=CA&q=Wi-Fi,WiFi&hl=en-GB

File: share/man/man4/wlan.4 [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
 .\" $FreeBSD$
 .\"
-.Dd January 26, 2021
+.Dd April 19, 2023
 .Dt WLAN 4
 .Os
 .Sh NAME
 .Nm wlan
-.Nd generic 802.11 link-layer support
+.Nd generic Wifi 802.11 link-layer support
```
Comment (bsdimp) [inline share/man/man4/wlan.4:33]: WiFi is yhe industry standard.

Comment (tomhukins) [reply share/man/man4/wlan.4]: Thank you for your helpful comments, @emaste and @bsdimp.  I have updated my pull request accordingly.

**Comment by tomhukins** [comment]
GitHub's UI mistakenly suggests 88406e6 forms part of the branch I submitted in this pull request, but most git clients show that it does not.  I suspect GitHub's software fails to cope with repositories whose canonical source lives elsewhere, in this case at git.freebsd.org.

**Comment by bsdimp** [comment]
I'll grab this and just commit the one relevant commit if my git client is confused.

**Comment by bsdimp** [comment]
Landed as 8fb0f07c7a13 ... I had to do this one by hand because of the confusion...

**Pull request closed without merging**